### PR TITLE
[Student][Teacher][Parent] Add aab support

### PR DESCRIPTION
**Pull request by CalvinKern**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1008537&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/student/src/main/java/com/instructure/student/util/AppManager.kt [hunk 1]
```diff
@@ -47,6 +48,10 @@ class AppManager : com.instructure.canvasapi2.AppManager(), AnalyticsEventHandli
     }
 
     override fun onCreate() {
+        if (MissingSplitsManagerFactory.create(this).disableAppIfMissingRequiredSplits()) {
+            // Skip app initialization.
+            return
```
Comment (joehoag) [inline apps/student/src/main/java/com/instructure/student/util/AppManager.kt]: So we basically just get a blank, unresponsive app if we have a missing required split?  Shouldn't we show a dialog explaining the situation or something like that?  (Maybe that's not possible without the missing split.)

Comment (CalvinKern) [reply apps/student/src/main/java/com/instructure/student/util/AppManager.kt]: This call will actually do that. It stops our application where we make the call, and shows a dialog alerting the user that they are missing components for the app. The dialog has a button to cancel and a button that will take them to the play store where they can install the missing pieces.

Comment (joehoag) [reply apps/student/src/main/java/com/instructure/student/util/AppManager.kt]: 👍

**Pull request merged**